lib/kubernixos.nix

```diff
@@ -34,6 +34,7 @@ let
           ${pkgs.kubeval}/bin/kubeval --strict -v ${cfg.version} \
               --output tab \
               --schema-location=file://${cfg.schemas} \
+              ${lib.concatMapStringsSep " " (kind: "--skip-kinds ${kind}") cfg.skipSchemas} \
```

Comment:
The docs on the flag says:
```
 --skip-kinds strings                    Comma-separated list of case-sensitive kinds to skip when validating against schemas
```

have you checked that this will actually work when giving a list with length > 1 ?

Reply:
I have tried many forms of comma-separated lists. None worked. I am not sure what the docs want here, but multiple --skip-kinds works fine.

Reply:
Fine by me then :D